.github/workflows/codecov.yml

```diff
@@ -16,10 +16,15 @@ jobs:
         with:
           go-version: '1.22'
           check-latest: true
-      - name: Optional init
-        run: make dependencies || true # Used in fortio for instance to prep for go tests
```

Comment:
May I assume dependencies is no longer needed, even for go test ?

Reply:
yes it's as commented was a fortio hack because:
https://github.com/fortio/fortio/blob/master/Makefile#L90
and similar to 
https://github.com/fortio/workflows/pull/47/files